src/components/transactions/GroupLabel/index.tsx

```diff
@@ -29,15 +35,17 @@ export const useGroupLabel = (item: Label): string => {
     return label
   }
 
-  const nextNonce = nonceInFuture ? safe.nonce : firstTx.transaction.executionInfo.nonce
```

Comment:
It looks like now we are always using `safe.nonce` in `futureNonceLabel`. Is the fallback to `firstTx.transaction.executionInfo.nonce` not needed anymore?

Reply:
Before we also always used `safe.nonce` but jut expressed in a unnecessary condition:
```ts  
const nonceInFuture = firstTx.transaction.executionInfo.nonce !== safe.nonce
const nextNonce = nonceInFuture ? safe.nonce : firstTx.transaction.executionInfo.nonce
```

which basically sets `nextNonce `to `safe.nonce` if the first transaction nonce is not equal to `safe.nonce` and if they are equal we set it to the first transaction nonce. But it doesn't change anything because they are equal.

Reply:
True, it should be equal. Nice catch! 💯